app.js

```diff
@@ -13,13 +14,45 @@ app.set('views', path.join(__dirname, 'views'));
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
-
+hbs.registerPartials(__dirname + '/views/partials')
 // ...
 
 // Add the route handlers here:
 
-app.get('/', (req, res) => {
-  res.render('index');
+app.get('/', (req, res) => res.render('index'));
+
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+
+      res.render('beers', { beers: beersFromApi })
+    })
+    .catch(error => console.log(error));
+
+});
+
+app.get('/beer-details/:id', (req, res) => {
+  console.log(req.params.id)
+  punkAPI
+    .getBeer()
+    .then(beerFromApi => {
+
+      res.render('beers', { beer: beerFromApi })
```

Comment:
Even though it's only one beer, the "beerFromApi" is still an array, so you need to actually pass `beersFromApi[0]`